src/component/BpmnVisualization.ts

```diff
@@ -92,4 +317,171 @@ export class BpmnVisualization {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_BACKGROUNDCOLOR, label.backgroundColor);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_BORDERCOLOR, label.borderColor);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_PADDING, label.padding);
+        }
+
+        const stroke = style.stroke;
+        if (stroke) {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_STROKECOLOR, stroke.color);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_STROKEWIDTH, stroke.width);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_STROKE_OPACITY, stroke.opacity);
+
+          if (stroke.dashPattern) {
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_DASHED, 1);
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_DASH_PATTERN, stroke.dashPattern);
+          }
+        }
+
+        const gradient = style.gradient;
+        if (gradient) {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_GRADIENTCOLOR, gradient.color);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_GRADIENT_DIRECTION, gradient.direction);
+        }
+
+        const image = style.image;
+        if (image) {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_IMAGE, image.url);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_IMAGE_WIDTH, image.width);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_IMAGE_HEIGHT, image.height);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_IMAGE_ALIGN, image.horizontalAlign);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_IMAGE_VERTICAL_ALIGN, image.verticalAlign);
+        }
+
+        if (this.isShapeStyleUpdate(style)) {
+          const fill = style.fill;
+          if (fill) {
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FILLCOLOR, fill.color);
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FILL_OPACITY, fill.opacity);
+
+            if (cellStyle.includes(ShapeBpmnElementKind.POOL) || cellStyle.includes(ShapeBpmnElementKind.LANE)) {
+              // Possible values are all HTML color names or HEX codes.
+              cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_SWIMLANE_FILLCOLOR, fill.color);
+            }
+          }
+
+          if (cellStyle.includes(ShapeBpmnElementKind.POOL) || cellStyle.includes(ShapeBpmnElementKind.LANE)) {
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_SEPARATORCOLOR, style.separatorColor);
```

Comment:
**chore:** 
The behavior is very weird. Dash on the separator, but we don"t choose to dash it.

❌ So, we decided to not implement it.